### DISCUSSION TITLE
请问如何同步配置

### DISCUSSION THREAD

**Discussion by JoeMob** [hoothin/UserScripts#419] [Q&A]
想在多台电脑上同步脚本配置，请问配置文件能不能直接复制出来然后覆盖其他电脑上的配置？

**Comment by hoothin** [comment]
这个每个脚本管理器都不一样吧，例如 tampermonkey 可以通过云盘同步。
如果你是指搜索酱，我有添加 webdav 支持。

**Reply by JoeMob** [reply]
Pagetual，我用Violentmonkey打开Sync script status同步之后，黑名单和翻页配置还是得手动调整。

**Reply by hoothin** [reply]
不好意思，我没用过暴力猴的同步功能，不过黑名单和翻页配置没有同步就说明脚本数据全都没有同步吧。建议你去暴力猴的社区搜索一下同步相关问题。我有印象似乎暴力猴的 github 上曾注明过不支持同步脚本数据。
